### PR TITLE
Shehz/account portal

### PR DESCRIPTION
**Pull request by msakbar**
Updates Made:
1. Added all Account Portal Pages (Overview, Getting Started, Custom Redirects, UserProfile/OrgProfile & Disable)
2. Added the contextual component references under Account Portal 
3. Added afterSignInURL, afterSignUpURL, redirectURL props to appropriate components
4. Added '"RedirectToOrganizationProfile "' page
5. Added '"RedirectToCreateOrganization "' page

### CODE REVIEW

File: docs/account-portal/overview.mdx [hunk 1]
```diff
@@ -4,62 +4,35 @@ description: The Account Portal offers a comprehensive solution for managing use
...
-</Callout>
+```sh
+<https://accounts>.[your-domain].com/sign-in
+<https://accounts>.[your-domain].com/sign-up
+<https://accounts>.[your-domain].com/user
+<https://accounts>.[your-domain].com/organization
+<https://accounts>.[your-domain].com/create-organization
+```
 
+You can access these links via the 'pages' tab under Account Portal in your Clerk dashboard.
```
Comment (perkinsjr) [inline docs/account-portal/overview.mdx:38]: Add links to dashboard here.

File: docs/account-portal/overview.mdx [hunk 2]
```diff
@@ -4,62 +4,35 @@ description: The Account Portal offers a comprehensive solution for managing use
...
-</Callout>
-
-## Custom redirects
-
 
-With the Account Portal, we default the user back to where they came from after a successful auth event (sign in or sign up) and pass this parameter via a `redirect_url` query param.
+# Why use the Account Portal?
 
-For situations where this may not be possible, Clerk defaults to your application homepage and expects the appropriate redirects will kick in. You can also specify default fallbacks directly in the dashboard as well. 
+The Account Portal provides all pages out of the box hosted on Clerk servers for you and requires minimal setup to get started, if you’re looking for the fastest way to get your application up and running, then this is a great choice.
```
Comment (perkinsjr) [inline docs/account-portal/overview.mdx:16]: This phrasing is very confusing. "provides all pages out of the box" what pages?

File: docs/account-portal/getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+    <Callout type="info" emoji="💡">
+     In this example, we are using `<RedirectToSignIn />`, but you can also use `<RedirectToSignUp />` to take customers directly to the ‘Sign Up’ page.
+    </Callout>
+
+</Steps>
+</Tab>
+<Tab>
+
+<Steps>
+## React
```
Comment (perkinsjr) [inline docs/account-portal/getting-started.mdx:165]: This should be outside the step

**Review by perkinsjr — CHANGES_REQUESTED** [review]
Just some changes to wording and some missed styling

File: docs/account-portal/custom-redirects.mdx [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/onboarding
+```
+</Tab>
+<Tab>
+```sh filename=".env" copy
+CLERK_AFTER_SIGN_IN_URL=/dashboard
+CLERK_AFTER_SIGN_UP_URL=/onboarding
+```
+</Tab>
+</Tabs>
```
Comment (alexisintech) [inline docs/account-portal/custom-redirects.mdx:27]: Let's use code block tabs here instead! This is my mistake - I had a different direction and was going to put some content into the Next.js tab but we went a different direction.

**Review by alexisintech — DISMISSED** [review]

File: docs/account-portal/direct-links.mdx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+## Fallback redirects
+
+In the case that a user does visit an Account Portal page directly without the query param, don't worry - Clerk offers a fallback configuration via the [Clerk Dashboard](https://dashboard.clerk.com).
+
+<Steps>
+
+### Set your static fallback host
+
+Set your fallback host under **Paths** in the Clerk Dashboard.
```
Comment (alexisintech) [inline docs/account-portal/direct-links.mdx:30]:
```suggestion
Set your fallback host under [Paths](https://dashboard.clerk.com/last-active?path=paths) in the Clerk Dashboard. 
```

File: docs/components/control/redirect-to-createorganization.mdx [hunk 6]
```diff
@@ -0,0 +1,87 @@
+---
+title: <RedirectToCreateOrganization />
+description: The `<RedirectToCreateOrganization />` component will navigate to the user profile URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```
Comment (alexisintech) [inline docs/components/control/redirect-to-createorganization.mdx:3]: Remove MDX syntax from frontmatter

```suggestion
description: The <RedirectToCreateOrganization /> component will navigate to the user profile URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```

File: docs/components/control/redirect-to-organizationprofile.mdx [hunk 7]
```diff
@@ -0,0 +1,87 @@
+---
+title: <RedirectToOrganizationProfile />
+description: The `<RedirectToOrganizationProfile />` component will navigate to the organization profile URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```
Comment (alexisintech) [inline docs/components/control/redirect-to-organizationprofile.mdx:3]: Remove MDX syntax from frontmatter

```suggestion
description: The <RedirectToOrganizationProfile /> component will navigate to the organization profile URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```

File: docs/components/control/redirect-to-signin.mdx [hunk 8]
```diff
@@ -3,8 +3,6 @@ title: <RedirectToSignIn />
 description: The `<RedirectToSignIn />` component will navigate to the sign in URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```
Comment (alexisintech) [inline docs/components/control/redirect-to-signin.mdx:3]: Remove MDX syntax from frontmatter

```suggestion
description: The <RedirectToSignIn /> component will navigate to the sign in URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```

File: docs/components/control/redirect-to-signup.mdx [hunk 9]
```diff
@@ -3,8 +3,6 @@ title: <RedirectToSignUp />
 description: The `<RedirectToSignUp />` component will navigate to the sign up URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```
Comment (alexisintech) [inline docs/components/control/redirect-to-signup.mdx:3]: Remove MDX syntax from frontmatter

```suggestion
description: The <RedirectToSignUp /> component will navigate to the sign up URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```

File: docs/components/control/redirect-to-userprofile.mdx [hunk 10]
```diff
@@ -3,14 +3,10 @@ title: <RedirectToUserProfile />
 description: The `<RedirectToUserProfile />` component will navigate to the user profile URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```
Comment (alexisintech) [inline docs/components/control/redirect-to-userprofile.mdx:3]: Remove MDX syntax from frontmatter

```suggestion
description: The <RedirectToUserProfile /> component will navigate to the user profile URL which has been configured in your application instance. The behavior will be just like a server-side (3xx) redirect, and will override the current location in the history stack.
```

File: docs/components/unstyled/sign-in-button.mdx [hunk 11]
```diff
@@ -3,10 +3,6 @@ title: <SignInButton />
 description: The `<SignInButton />` component is a button that links to the sign-in page or displays the sign-in modal.
```
Comment (alexisintech) [inline docs/components/unstyled/sign-in-button.mdx:3]: Remove MDX syntax from frontmatter

```suggestion
description: The <SignInButton /> component is a button that links to the sign-in page or displays the sign-in modal.
```

File: docs/components/unstyled/sign-up-button.mdx [hunk 12]
```diff
@@ -3,10 +3,6 @@ title: <SignUpButton />
 description: The `<SignUpButton />` component is a button that links to the sign-up page or displays the sign-up modal.
```
Comment (alexisintech) [inline docs/components/unstyled/sign-up-button.mdx:3]: Remove MDX syntax from frontmatter

```suggestion
description: The <SignUpButton /> component is a button that links to the sign-up page or displays the sign-up modal.
```

File: docs/account-portal/custom-redirects.mdx [hunk 13]
```diff
@@ -0,0 +1,188 @@
+---
+title: Customizing your redirects | Account Portal
+description:  Customize where your users are redirected to after they sign-in or sign-up.
```
Comment (alexisintech) [inline docs/account-portal/custom-redirects.mdx:3]:
```suggestion
description:  Customize where your users are redirected to after they sign in or sign up.
```

File: docs/account-portal/direct-links.mdx [hunk 14]
```diff
@@ -0,0 +1,54 @@
+---
+title: Direct links | Account Portal
+description: Learn how to share direct links to your Account Portal pages, and how to set up fallback redirects
```
Comment (alexisintech) [inline docs/account-portal/direct-links.mdx:3]:
```suggestion
description: Learn how to share direct links to your Account Portal pages, and how to set up fallback redirects.
```

**Review by alexisintech — CHANGES_REQUESTED** [review]
Just really nit picky things because I'm crazy, but with the new OG Images about to ship, we want to remove MDX syntax from front matter - its a ticket that I am going to get to after the OG Images ship.

**Review by alexisintech — APPROVED** [review]
😸💖

**Pull request merged**